packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -87,6 +90,9 @@ async function clientConfig(env) {
 		},
 
 		plugins: [
+			...(!isProd && refresh
+				? [new webpack.HotModuleReplacementPlugin(), new RefreshPlugin()]
```

Comment:
are both of these required together?

Reply:
Yes `HotModuleReplacementPlugin ` enables `module.hot` to opt-out of page-reload